Author: Roland Rosenfeld <roland@debian.org>
Description: Stop converting 8bit chars in the documentation (#203697)
Bug-Debian: http://bugs.debian.org/203697

--- a/doc/source/ldp.dsl.in
+++ b/doc/source/ldp.dsl.in
@@ -47,9 +47,9 @@
 
 ;; this is necessary because right now jadetex does not understand
 ;; symbolic entities, whereas things work well with numeric entities.
-(declare-characteristic preserve-sdata?
-  "UNREGISTERED::James Clark//Characteristic::preserve-sdata?"
-  #f)
+;(declare-characteristic preserve-sdata?
+;  "UNREGISTERED::James Clark//Characteristic::preserve-sdata?"
+;  #f)
 
 ;; put the legal notice in a separate file
 (define %generate-legalnotice-link%
@@ -257,9 +257,9 @@
 
 ;; this is necessary because right now jadetex does not understand
 ;; symbolic entities, whereas things work well with numeric entities.
-(declare-characteristic preserve-sdata?
-  "UNREGISTERED::James Clark//Characteristic::preserve-sdata?"
-  #f)
+;(declare-characteristic preserve-sdata?
+;  "UNREGISTERED::James Clark//Characteristic::preserve-sdata?"
+;  #f)
 
 ;; put the legal notice in a separate file
 (define %generate-legalnotice-link%
