From: Roland Rosenfeld <roland@debian.org>
Date: Fri, 05 Jun 2020 15:51:09 +0200
Subject: Several Debian specific changes to config file

--- a/config
+++ b/config
@@ -127,7 +127,7 @@
 #          config file, because it is used while the config file is
 #          being read.
 #
-#user-manual https://www.privoxy.org/user-manual/
+user-manual /usr/share/doc/privoxy/user-manual
 #
 #  1.2. trust-info-url
 #  ====================
@@ -259,7 +259,7 @@
 #
 #      No trailing "/", please.
 #
-confdir .
+confdir /etc/privoxy
 #
 #  2.2. templdir
 #  ==============
@@ -344,7 +344,7 @@ confdir .
 #
 #      No trailing "/", please.
 #
-logdir .
+logdir /var/log/privoxy
 #
 #  2.5. actionsfile
 #  =================
@@ -2474,7 +2474,7 @@ socket-timeout 300
 #
 #      ca-directory /usr/local/etc/privoxy/CA
 #
-#ca-directory /usr/local/etc/privoxy/CA
+#ca-directory /etc/privoxy/CA
 #
 #  7.2. ca-cert-file
 #  ==================
@@ -2615,7 +2615,7 @@ socket-timeout 300
 #
 #      certificate-directory /usr/local/var/privoxy/certs
 #
-#certificate-directory /usr/local/var/privoxy/certs
+#certificate-directory /var/lib/privoxy/certs
 #
 #  7.6. trusted-cas-file
 #  ======================
@@ -2648,7 +2648,7 @@ socket-timeout 300
 #
 #      trusted-cas-file trusted_cas_file.pem
 #
-#trusted-cas-file trustedCAs.pem
+#trusted-cas-file /etc/ssl/certs/ca-certificates.crt
 #
 #  8. WINDOWS GUI OPTIONS
 #  =======================
