From: Roland Rosenfeld <roland@debian.org>
Date: Sat, 11 Feb 2006 21:27:14 +0100
Subject: Several Debian specific changes to config file

--- a/config
+++ b/config
@@ -126,7 +126,7 @@
 #          config file, because it is used while the config file is
 #          being read.
 #
-#user-manual https://www.privoxy.org/user-manual/
+user-manual /usr/share/doc/privoxy/user-manual
 #
 #  1.2. trust-info-url
 #  ====================
@@ -258,7 +258,7 @@
 #
 #      No trailing "/", please.
 #
-confdir .
+confdir /etc/privoxy
 #
 #  2.2. templdir
 #  ==============
@@ -343,7 +343,7 @@ confdir .
 #
 #      No trailing "/", please.
 #
-logdir .
+logdir /var/log/privoxy
 #
 #  2.5. actionsfile
 #  =================
