From: Roland Rosenfeld <roland@debian.org>
Date: Sat, 19 Sep 2015 15:05:41 +0200
Subject: Use Debian docbook2man-spec.pl (from docbook-utils) instead of local
 copy, to make package reproducible.
Forwarded: not-needed

--- a/GNUmakefile.in
+++ b/GNUmakefile.in
@@ -539,7 +539,7 @@ dok-man:
 # target for man page generation!
 man: dok-release
 	mkdir -p doc/source/temp && cd doc/source/temp && $(RM) * ;\
-	$(NSGMLS) ../privoxy-man-page.sgml  | sgmlspl ../../../utils/docbook2man/docbook2man-spec.pl &&\
+	$(NSGMLS) ../privoxy-man-page.sgml  | sgmlspl /usr/share/perl5/sgmlspl-specs/docbook2man-spec.pl &&\
 	perl -pi.bak -e 's/ <URL:.*>//; s/\[ /\[/g' $(MAN_PAGE) ;\
 	perl -pi.bak -e "s/\[ /\[/g;s/á/\\\\['a]/g;s/é/\\\\['e]/g" $(MAN_PAGE); \
 	perl -pi.bak -e "s/ö/\\\\[:o]/g" $(MAN_PAGE); \
