From: Roland Rosenfeld <roland@debian.org>
Date: Tue, 01 Jan 2019 17:33:42 +0100
Subject: Comment trusted-cgi-referer pointing to example.org

--- a/config
+++ b/config
@@ -1215,7 +1215,7 @@ enable-proxy-authentication-forwarding 0
 #      |the user's knowledge.                                |
 #      +-----------------------------------------------------+
 #
-trusted-cgi-referer http://www.example.org/
+#trusted-cgi-referer http://www.example.org/
 #
 #
 #  5. FORWARDING
