From: Roland Rosenfeld <roland@debian.org>
Date: Wed, 11 Jan 2017 22:24:55 +0100
Subject: unblock adventofcode.com
Bug-Debian: https://bugs.debian.org/848211
Forwarded: https://sourceforge.net/p/ijbswa/actionsfile-feedback/937/

--- a/default.action.master
+++ b/default.action.master
@@ -1941,7 +1941,8 @@ klikki.fi/adv/
 #MASTER# UNBLOCK-REFERRER: http://www.ardmediathek.de/tv
 # URL = http://www.ardmediathek.de/ard/static/img/ardmt/banner/ard-mediathek.svg
 .ardmediathek.de/
-
+#MASTER# UNBLOCK-REFERRER: http://adventofcode.com
+adventofcode.com
 
 #############################################################################
 # Site-specific special rules:
