From: Fabian Keil <fk@fabiankeil.de>
Date: Fri, 4 Jan 2019 14:03:14 +0000
Subject: Update a bunch of tests that have to expect status code 403 now
Origin: https://www.privoxy.org/gitweb/?p=privoxy.git;a=commitdiff_plain;h=HEAD;hp=10a83542e5fba75cd6da2e08ad461ac7439ba807

--- a/regression-tests.action
+++ b/regression-tests.action
@@ -722,48 +722,70 @@ TAG:^Proxy-Authorization header forwardi
 # Expect Status Code = 403
 # Level = 21
 # Fetch Test = http://p.p/edit-actions
+# Expect Status Code = 403
 # Level = 12
 # Fetch Test = http://p.p/eaa
+# Expect Status Code = 403
 # Level = 12
 # Fetch Test = http://p.p/eau
+# Expect Status Code = 403
 # Level = 12
 # Fetch Test = http://p.p/ear
+# Expect Status Code = 403
 # Level = 12
 # Fetch Test = http://p.p/eal
+# Expect Status Code = 403
 # Level = 12
 # Fetch Test = http://p.p/eafu
+# Expect Status Code = 403
 # Level = 12
 # Fetch Test = http://p.p/eas
+# Expect Status Code = 403
 # Level = 12
 # Fetch Test = http://p.p/easa
+# Expect Status Code = 403
 # Level = 12
 # Fetch Test = http://p.p/easr
+# Expect Status Code = 403
 # Level = 12
 # Fetch Test = http://p.p/eass
+# Expect Status Code = 403
 # Level = 12
 # Fetch Test = http://p.p/edit-actions-for-url
+# Expect Status Code = 403
 # Level = 12
 # Fetch Test = http://p.p/edit-actions-list
 # Level = 12
 # Fetch Test = http://p.p/edit-actions-submit
+# Expect Status Code = 403
 # Level = 12
+# Expect Status Code = 403
 # Fetch Test = http://p.p/edit-actions-url
+# Expect Status Code = 403
 # Level = 12
 # Fetch Test = http://p.p/edit-actions-url-form
+# Expect Status Code = 403
 # Level = 12
 # Fetch Test = http://p.p/edit-actions-add-url
+# Expect Status Code = 403
 # Level = 12
 # Fetch Test = http://p.p/edit-actions-add-url-form
+# Expect Status Code = 403
 # Level = 12
 # Fetch Test = http://p.p/edit-actions-remove-url
+# Expect Status Code = 403
 # Level = 12
 # Fetch Test = http://p.p/edit-actions-remove-url-form
+# Expect Status Code = 403
 # Level = 12
 # Fetch Test = http://p.p/edit-actions-section-add
+# Expect Status Code = 403
 # Level = 12
 # Fetch Test = http://p.p/edit-actions-section-remove
+# Expect Status Code = 403
 # Level = 12
 # Fetch Test = http://p.p/edit-actions-section-swap
+# Expect Status Code = 403
 # Level = 12
 # Fetch Test = http://p.p/error-favicon.ico
 # Fetch Test = http://p.p/favicon.ico
