From: Roland Rosenfeld <roland@debian.org>
Subject: Add new features to show-status page.
Date: Sun, 06 Dec 2020 14:14:43 +0100
Forwarded: https://www.privoxy.org/gitweb/?p=privoxy.git;a=patch;h=d83b7ce5

--- a/templates/show-status
+++ b/templates/show-status
@@ -299,6 +299,19 @@
               </td>
             </tr>
             <tr>
+              <td><code>FEATURE_DYNAMIC_PCRE</code></td>
+              <td>@if-FEATURE_DYNAMIC_PCRE-then@ Yes @else-not-FEATURE_DYNAMIC_PCRE@ No @endif-FEATURE_DYNAMIC_PCRE@</td>
+              <td>Dynamically link to the PCRE library. This is set automatically
+               by <code>./configure</code> if you do not have libpcre installed.
+               Dynamically linking to an external libpcre is recommended as the one that is distributed
+               with Privoxy itself is outdated and lacks various features and bug-fixes you may be interested in.</td>
+            </tr>
+            <tr>
+              <td><code>FEATURE_EXTENDED_STATISTICS</code></td>
+              <td>@if-FEATURE_EXTENDED_STATISTICS-then@ Yes @else-not-FEATURE_EXTENDED_STATISTICS@ No @endif-FEATURE_EXTENDED_STATISTICS@</td>
+              <td>Gather statistics for block reasons and filter executions.</td>
+            </tr>
+            <tr>
               <td><code>FEATURE_EXTERNAL_FILTERS</code></td>
               <td>@if-FEATURE_EXTERNAL_FILTERS-then@ Yes @else-not-FEATURE_EXTERNAL_FILTERS@ No @endif-FEATURE_EXTERNAL_FILTERS@</td>
               <td>
@@ -308,14 +321,6 @@
               </td>
             </tr>
             <tr>
-              <td><code>FEATURE_DYNAMIC_PCRE</code></td>
-              <td>@if-FEATURE_DYNAMIC_PCRE-then@ Yes @else-not-FEATURE_DYNAMIC_PCRE@ No @endif-FEATURE_DYNAMIC_PCRE@</td>
-              <td>Dynamically link to the PCRE library. This is set automatically
-               by <code>./configure</code> if you do not have libpcre installed.
-               Dynamically linking to an external libpcre is recommended as the one that is distributed
-               with Privoxy itself is outdated and lacks various features and bug-fixes you may be interested in.</td>
-            </tr>
-            <tr>
               <td><code>FEATURE_FAST_REDIRECTS</code></td>
               <td>@if-FEATURE_FAST_REDIRECTS-then@ Yes @else-not-FEATURE_FAST_REDIRECTS@ No @endif-FEATURE_FAST_REDIRECTS@</td>
               <td>Allows the +fast-redirects action, to bypass redirect and logging scripts.</td>
@@ -353,6 +358,14 @@
               </td>
             </tr>
             <tr>
+              <td><code>FEATURE_PCRE_HOST_PATTERNS</code></td>
+              <td>@if-FEATURE_PCRE_HOST_PATTERNS-then@ Yes @else-not-FEATURE_PCRE_HOST_PATTERNS@ No @endif-FEATURE_PCRE_HOST_PATTERNS@</td>
+              <td>Allow to use extended host patterns and vanilla host patterns
+               at the same time by prefixing extended host patterns with
+               "PCRE-HOST-PATTERN:".
+              </td>
+            </tr>
+            <tr>
               <td><code>FEATURE_NO_GIFS</code></td>
               <td>@if-FEATURE_NO_GIFS-then@ Yes @else-not-FEATURE_NO_GIFS@ No @endif-FEATURE_NO_GIFS@</td>
               <td>Use PNG instead of GIF for the built-in images.</td>
--- a/cgisimple.c
+++ b/cgisimple.c
@@ -2097,6 +2097,22 @@ static jb_err show_defines(struct map *e
 #else
          0,
 #endif
+      },
+      {
+         "FEATURE_EXTENDED_STATISTICS",
+#ifdef FEATURE_EXTENDED_STATISTICS
+         1,
+#else
+         0,
+#endif
+      },
+      {
+         "FEATURE_PCRE_HOST_PATTERNS",
+#ifdef FEATURE_PCRE_HOST_PATTERNS
+         1,
+#else
+         0,
+#endif
       }
    };
 
